keyboards/romac/config.h

```diff
@@ -0,0 +1,57 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    qmkbuilder
```

Comment:
Your name should go here.

```suggestion
#define MANUFACTURER    TheRoyalSweatshirt
```